upstream-community-operators/knative-serving-operator/0.13.0/knative-serving-operator.v0.13.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,205 @@
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: knative-serving-operator
+                - name: SYSTEM_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: METRICS_DOMAIN
+                  value: knative.dev/serving-operator
+                - name: CONFIG_LOGGING_NAME
+                  value: config-logging
+                - name: CONFIG_OBSERVABILITY_NAME
+                  value: config-observability
+                image: gcr.io/knative-releases/knative.dev/serving-operator/cmd/manager@sha256:fe16db1ec4f35aaf2e70cb761eb25604846f3e434d2d4e161a2fd23e005f2dfd
+                imagePullPolicy: IfNotPresent
+                name: knative-serving-operator
+                ports:
+                - containerPort: 9090
+                  name: metrics
+                resources: {}
+              serviceAccountName: knative-serving-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - knative-serving-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - operator.knative.dev
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - '*'
+        serviceAccountName: knative-serving-operator
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - serverless
+  - scale to zero
+  links:
+  - name: Documentation
+    url: https://www.knative.dev/docs/serving/
+  - name: Source Repository
+    url: https://github.com/knative/serving
+  maintainers:
+  - email: knative-dev@googlegroups.com
+    name: The Knative Authors
+  maturity: alpha
+  minKubeVersion: 1.15.0
+  provider:
+    name: The Knative Authors
+  replaces: knative-serving-operator.v0.12.2
```

Comment:
What do we we define here?
Is it the previous version we are about to overwrite?

Reply:
yeah. that's true.
I wouldn't call it "overwrite". But, it is more like OLM is going to update from 0.12.2 version of the operator to new version.